cgi-bin/Apache/LiveJournal.pm

```diff
@@ -221,7 +221,7 @@ sub send_concat_res_response {
 
     # Might contain cache buster "?v=3234234234" at the end;
     # plus possibly other unique args (caught by the .*)
-    $args =~ s/\?v=\d+.*$//;
+    $args =~ s/\?v=\d*.*$//;
```

Comment:
Was this also supposed to allow ?v=non-digit-values? If so, the whole \d part is no longer needed.

Reply:
Well, I don't have any real use for specifying non-digit values. But good point -- if we're not actually USING the numbers for anything and are just stripping the params off to throw them away, then why check for them at all?

```suggestion
    $args =~ s/\?v=.*$//;
```